torch_tvm/operators.cpp

```diff
@@ -746,7 +785,7 @@ RegisterTVMOperator reg({
        attrs->newshape = relayToArray<tvm::Integer>(inputs[1]);
        TORCH_INTERNAL_ASSERT(attrs->newshape.size() > 0);
        if (static_cast<int64_t>(attrs->newshape[0]) != -1) {
-	       std::cerr << "WARNING: reshape with -1 as the first value has known incompatibility with PyTorch semantics.\n";
+         std::cerr << "WARNING: reshape with -1 as the first value has known incompatibility with PyTorch semantics.\n";
```

Comment:
when you change the std::cout above to the logging framework, mind changing this one as well?

Reply:
will upd